src/fairseq2/recipes/lm/_online_finetune/_rewards.py

```diff
@@ -338,14 +338,18 @@ def __init__(self, gangs, reward_model, answer_key, prompt_key, tokenizer):
         self.tokenizer = AutoTokenizer.from_pretrained(tokenizer)
 
     def wrap_text(self, prompt_text, rollout_text):
-        wrapped_text = [
-            {"role": "user", "content": prompt_text},
-            {"role": "assistant", "content": rollout_text},
+        messages = [
+            {
+                "role": "user",
+                "content": prompt_text,
+            },
+            {
+                "role": "assistant",
+                "content": rollout_text,
+            },
         ]
```

Comment:
note that athene-rm will only work with HF now. It's 2x faster than vllm, so I don't think we need vllm for it